Dockerfile.updater-core

```diff
@@ -90,14 +90,14 @@ WORKDIR $DEPENDABOT_HOME/dependabot-updater
 
 # Install Ruby from official Docker image
 # When bumping Ruby minor, need to also add the previous version to `bundler/helpers/v{1,2}/monkey_patches/definition_ruby_version_patch.rb`
-COPY --from=ruby:3.1.4-bullseye --chown=dependabot:dependabot /usr/local /usr/local
+COPY --from=ruby:3.1.4-bookworm --chown=dependabot:dependabot /usr/local /usr/local
```

Comment:
We could base the image off of the ruby image and delete this line, and then get Dependabot bumps for it.

Reply:
It's an interesting idea for sure, but I'd rather do it after this PR lands. I considered switching to `--from=ruby:3.1.4` since today it's the same as `ruby:3.1.4-bookworm` but we used to do that and ran into problems when it switched its base debian image under our feet to one that conflicted with our version of Ubuntu used elsewhere... it'd probably be okay if it were the base image, but still let's postpone looking at that to a later PR and here just catch us up to a newer Ubuntu.